src/AstarteManager.tsx

```diff
@@ -85,8 +85,11 @@ function loadSession(): Session | null {
   } catch {
     session = null;
   }
-  if (_.get(session, '_version') === SESSION_VERSION) {
-    return _.omit(session, '_version');
+  if (_.get(session, '_version') === SESSION_VERSION && session) {
```

Comment:
we should check for session being null before using it, making use of the `&&` shortcircuit
```suggestion
  if (session && _.get(session, '_version') === SESSION_VERSION) {
```